### PR TITLE
add log if remove device MarshalMembershipUpdate error

### PR DESCRIPTION
**Pull request by jidalong**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
add log if remove device MarshalMembershipUpdate error

**Does this PR introduce a user-facing change?**:
NONE

```release-note

```

### CODE REVIEW

File: edge/pkg/devicetwin/dtmanager/membership.go [hunk 1]
```diff
@@ -295,7 +295,8 @@ func removeDevice(context *dtcontext.DTContext, toRemove []dttype.Device, baseMe
 		deleteResult := dttype.MembershipUpdate{BaseMessage: baseMessage, RemoveDevices: RemoveDevices}
 		result, err := dttype.MarshalMembershipUpdate(deleteResult)
 		if err != nil {
-
+			klog.Errorf("Remove device %s failed, marshal membership err: %s", device.ID, err)
+			continue
 		} else {
```
Comment (zc2638) [inline edge/pkg/devicetwin/dtmanager/membership.go:300]: Looks like we don't need `else` to judge

Comment (gy95) [reply edge/pkg/devicetwin/dtmanager/membership.go:300]: yes, just remove else

Comment (jidalong) [reply edge/pkg/devicetwin/dtmanager/membership.go]: Done

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**